frontend/src/App.tsx

```diff
@@ -50,9 +74,62 @@ function App() {
               onClick: () => navigate("/settings"),
             },
             {
-              children: <Outline.ArrowRightOnRectangleIcon />,
+              children: (
```

Comment:
I think we want to go for the dropdown approach here as it feels like the more suitable components, we can reuse your IconInitials component pass it in the label of the dropdown, build the content of the popup in the children (dropdown content).